blocks/marketo/marketo.css

```diff
@@ -0,0 +1,336 @@
+/* stylelint-disable selector-id-pattern */
+/* stylelint-disable selector-class-pattern */
+.marketo {
+    width: 100%;
+}
+
+/* Merative styling for Marketo form */
+.mktoForm {
+    margin: 0 auto !important;
+    width: 100% !important
+}
+
+.mktoForm .mktoOffset {
+    width: 0 !important
+}
+
+.mktoForm .mktoFieldWrap .mktoHtmlText {
+    color: #20201f !important;
```

Comment:
This is not a standard color from the Merative palette?

Reply:
updated to variable